test/util/testnode/read.go

```diff
@@ -39,6 +39,63 @@ func ReadBlockchain(ctx context.Context, rpcAddress string) ([]*types.Block, err
 	return ReadBlockHeights(ctx, rpcAddress, 1, status.SyncInfo.LatestBlockHeight)
 }
 
+// ReadBlockchainHeaders retrieves the blockchain headers from height 1 up to
+// latest available height from the node at rpcAddress and returns it.
+// The headers are returned in ascending order (lowest first).
+func ReadBlockchainHeaders(ctx context.Context, rpcAddress string) ([]*types.BlockMeta, error) {
```

Comment:
[nit] this function name is a bit misleading b/c `Header` is a distinct struct in celestia-core. This function returns a list of `BlockMeta` so IMO it could be:

```suggestion
func ReadBlockMetas(ctx context.Context, rpcAddress string) ([]*types.BlockMeta, error) {
```